homeassistant/components/fujitsu_hvac/climate.py

```diff
@@ -0,0 +1,205 @@
+"""Support for Fujitsu HVAC devices that use the Ayla Iot platform."""
+from asyncio import gather
+from contextlib import suppress
+from typing import Any
+
+from ayla_iot_unofficial import AylaAuthError
+from ayla_iot_unofficial.fujitsu_hvac import Capability, FujitsuHVAC
+
+from homeassistant.components.climate import (
+    ClimateEntity,
+    ClimateEntityFeature,
+    HVACMode,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import ATTR_TEMPERATURE, PRECISION_HALVES, UnitOfTemperature
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.device_registry import DeviceInfo
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .const import (
+    API,
+    DOMAIN,
+    FUJI_TO_HA_FAN,
+    FUJI_TO_HA_HVAC,
+    FUJI_TO_HA_SWING,
+    HA_TO_FUJI_FAN,
+    HA_TO_FUJI_HVAC,
+    HA_TO_FUJI_SWING,
+)
+
+
+async def async_setup_entry(
+    hass: HomeAssistant,
+    entry: ConfigEntry,
+    async_add_entities: AddEntitiesCallback,
+) -> None:
+    """Set up one Fujitsu HVAC device."""
+    api = hass.data[DOMAIN][API]
+
+    devices = await api.async_get_devices()
+    devices = list(filter(lambda x: isinstance(x, FujitsuHVAC), devices))
+    await gather(*[dev.async_update() for dev in devices])
```

Comment:
Will you add more platforms in the future? Did you consider a coordinator?

Reply:
I might add a Sensor to represent the outside temperature at some point. I did consider a coordinator. I might switch to that in the future.